### DISCUSSION TITLE
Actividad 1 - Grupo 2

### DISCUSSION THREAD

**Discussion by BenjaminPeres596** [frlp-utn-ingsoft/catedra#91] [Show and tell]
https://github.com/JuaniLezcano/vetsoft

**Comment by ivanduva** [comment]
Buenas!
Chicos, está rota la funcionalidad de editar medicina. Les pedimos que lo arreglen para aprobar la actividad. Además en el form no aparecen los mensajes de error al validar descripción y dosis. Este problema es menor pero estaría bueno que lo arreglen también. Avisen si tienen problemas para identificar donde está el problema. Nosotros ya lo identificamos. Debería arreglarlo el integrante que hizo esa funcionalidad.
Por otro lado, les dejamos algunos comentarios para las próximas actividades:

- Los branches tienen nombres todos distintos. Estaría bueno que lo vayan estandarizando. Hay uno que tiene el nombre de un integrante, lo cual es redundante porque los commits que se van haciendo en ese branch ya contienen el nombre de usuario de quien los hace. Lo mismo con los commits, vimos algunos mensajes de commit entre corchetes y no se entiende muy bien por qué.
- Vimos algunos casos en los que repiten los nombres de commit con respecto al commit anterior. Por ejemplo: https://github.com/JuaniLezcano/vetsoft/commit/52a6e5b64c6b338b5f1cd1b8b46df0b23c6af636 y https://github.com/JuaniLezcano/vetsoft/commit/1ae09b2e84ecba556619b1bede3444c3a23122d7. Esto puede ser porque resolvieron conflictos y el commit de merge les quedó con el nombre del commit anterior. Traten de dejar el mensaje default de git porque sino es difícil de entender.
- Hay algunos commits poco atómicos, por ejemplo este https://github.com/JuaniLezcano/vetsoft/commit/351719d85f10f0ee8d48fb92e0ec917f0deeec57 resuelve todo el ABM en un mismo commit, desde las views hasta los templates.
- @ValentinMilocco fijate que hiciste primero el commit de las urls, después el de las vistas y último el del modelo, cuando tenía que ser exactamente al revés. Entendemos que lo hiciste todo junto y luego commiteaste por separado, pero el problema de que haya quedado así es que si te movés al commit de las urls, la app se rompe porque no existen las views. Lo mismo si te movés al de las views, se rompe porque no existe el modelo. La app debería funcionar en todos los commits, tengan cuidado con eso la próxima.

Saludos!

**Reply by PeresValentin** [reply]
Profe! Ya están publicados los cambios en los errores que mencionaste.
Saludos

**Comment by ivanduva** [comment]
Perfecto, queda aprobado entonces.
Saludos
